robocop/checkers/lengths.py

```diff
@@ -344,11 +334,7 @@ class NumberOfReturnedArgsChecker(VisitorChecker):
     def visit_Keyword(self, node):  # noqa
         self.generic_visit(node)
 
-    def visit_ForLoop(self, node):  # noqa
-        self.generic_visit(node)
-
-    def visit_For(self, node):  # noqa
-        self.generic_visit(node)
+    visit_For = visit_ForLoop = visit_Keyword
```

Comment:
Check if it works with visit_While, visit_Try and also if we need to visit_For/visit_ForLoop at all

Reply:
No need it for them - I will remove it.